app/javascript/controllers/index.js

```diff
@@ -2,28 +2,16 @@
 // Run that command whenever you add a new controller or create them with
 // ./bin/rails generate stimulus controllerName
 
-import { application } from "./application"
+import { application } from "./application";
 
+import EditEventController from "./edit_event_controller";
+application.register("edit-event", EditEventController);
 
-import HelloController from "./hello_controller"
-application.register("hello", HelloController)
+import EventsController from "./events_controller";
+application.register("events", EventsController);
 
-import MapController from "./map_controller"
-application.register("map", MapController)
+import HelloController from "./hello_controller";
+application.register("hello", HelloController);
 
-
```

Comment:
You cannot delete these as they are all the JS controllers for the itinerary show